pkg/docker/names.go

```diff
@@ -31,7 +31,7 @@ func FilePathToImageName(fileName string) (imageName string, err error) {
 		dirL := len(imageDirName)
```

Comment:
does this section of code just want the filename stripping away the dir? ... or i'm i reading wrong .. you can just use `filepath.Base(filename)` if so

Reply:
@gambol99 - totally, must have done this with headphones off - updated.